client/controllers/public/home.js

```diff
@@ -1,9 +1,59 @@
 Template.home.onCreated(function() {
-  Meteor.subscribe('latestChannels');
+  var self = this;
+  self.autorun(function() {
+    Meteor.subscribe('latestChannels');
+    if (Meteor.userId()) {
+      Meteor.subscribe('myChannels');
+    }
```

Comment:
What would be the benefit of using a template variable? I'm not sure I understand the context.